view/widgets/acquisition_widgets/volume_model.py

```diff
@@ -161,6 +165,34 @@ def set_path_pos(self, coord_order: list):
             self.start.setVisible(False)
             self.end.setVisible(False)
 
+    def add_fov_image(self, image: np.array, coords: list):
+        """add image to model assuming image has same fov dimensions and orientation
+        :param image: numpy array of image to display in model
+        :param coords: list of coordinates corresponding to the coordinate plane of model"""
+
+        image_rgba = makeRGBA(image, levels=[0, 256])[0]
+
+        image_rgba[:, :, 3] = 200
+
+        gl_image = GLImageItem(image_rgba,
```

Comment:
Here I would pass in the raw uint16 image and set the min/max brightness of how it is displayed through I think the _levels_ input arg. https://pyqtgraph.readthedocs.io/en/latest/api_reference/graphicsItems/imageitem.html_